resources/testharness.js

```diff
@@ -4754,7 +4754,7 @@
         if ('META_TITLE' in global_scope && META_TITLE) {
             return META_TITLE;
         }
-        if ('location' in global_scope) {
+        if ('location' in global_scope && location.pathname) {
```

Comment:
I feel like this would fit in more logically:

```suggestion
        if ('location' in global_scope && 'pathname' in location) {
```

And from the sound of it, you just have it _not present_ rather than falsey?

Reply:
Yeah, not present.